components/yearly_monthly_toggle/index.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+
+import SwitchSelector from 'react-switch-selector';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import './index.scss';
+
+interface Props {
+    updatePrice: (isMonthly: boolean) => void;
+}
+
+function YearlyMonthlyToggle({updatePrice}: Props) {
+    const [isMonthly, setIsMonthly] = useState(true);
+    const [toggleBorderClassName, setToggleBorderClassName] = useState('toggle-border');
+
+    const monthlyLabel = 'Monthly';
+    const yearlyLabel = 'Yearly';
```

Comment:
We should include these texts to the translations files

Reply:
Cool, I'll make those changes :)